sf_stats.Api/sf_stats.DataAccess.MSSql/EntityConfigurations/GameEntityConfiguration.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using sf_stats.Domain.Entities;
+using System;
+
+
+namespace sf_stats.DataAccess.MSSql.EntityConfigurations
+{
+    public class GameEntityConfiguration : IEntityTypeConfiguration<Game>
+    {
+        public void Configure(EntityTypeBuilder<Game> builder)
+        {
+            builder.HasKey(x => x.Id);
+            builder.HasOne(x => x.Season).WithMany(x => x.Games);
+            builder.HasOne(x => x.Division).WithMany(x => x.Games);
+            builder.HasOne(x => x.Home_Team).WithMany(x => x.HomeGames).OnDelete(DeleteBehavior.NoAction);
+            builder.HasOne(x => x.Away_Team).WithMany(x => x.AwayGames).OnDelete(DeleteBehavior.NoAction);
```

Comment:
Not sure what delete behavior we want here, I just went with no action to get things working

Reply:
apparently EF Core sees it as a cyclical update issue so it has to be NoCascase